### PR TITLE
Adding jihssan's coordinants

### LINKED ISSUE #14044: Please add me - Reims, France
Hello! :wave: Please add me as a collaborator to this repo. 

City: Reims 
Country: France
Longitude: 4.031696
Latitude: 49.258329

### PR DESCRIPTION
**Pull request by jihssan**
@githubteacher What do you think?

This pull request closes #14044.

### CODE REVIEW

**Comment by Rohit-Wanderer** [comment]
hello

**Comment by jihssan** [comment]
It s seems containing some errors.
What should i do?

**Comment by jihssan** [comment]
@Rohit-Wanderer  It seems containing some errors !!! isnt it?

File: _pins/jihssan.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: wndpof
```
Comment (jihssan) [inline _pins/jihssan.json]: I shouldnt put my username?

Comment (je-chelsea) [reply _pins/jihssan.json]: yeah, you should (: 👍

Comment (mvicendese) [inline _pins/jihssan.json]: Just dropping in to say hi.

**Comment by mvicendese** [comment]
@githubteacher  did you see my changes?

**Comment by antarmukhee** [comment]
Looking great! :)

File: _pins/jihssan-Reims.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+user: windpof
```
Comment (antarmukhee) [inline _pins/jihssan-Reims.json]: welcome

**Comment by Ofega** [comment]
@jihssan Hello!!! 👍

File: _pins/jihssan.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: jihssan
+latitude: 49.248092
```
Comment (toon159) [inline _pins/jihssan.json]: I hate this case sensitive so much :grin:

**Pull request merged**